app/src/main/java/com/acmvit/androidchallenge1/Activities/MainActivity.java

```diff
@@ -12,6 +15,16 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        Button bDhanesh=(Button)findViewById(R.id.btn_dhanesh);
```

Comment:
give meaning full names to your variables.

Don't give variable names like bDhanesh